eZ/Bundle/EzPublishCoreBundle/Resources/views/default/embed.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for field in content.FieldsByLanguage %}
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
If somebody has a suggestion for the contents of the embed template, I also accepts PRs to this branch (for @crevillo) or commits to the branch (@dev-team) ;-)

We could get away with a template that displays the name, with a link to the embedded content full view.